### PR TITLE
New Installer plugins bugs

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for New Issue.
#### Summary of Changes

This PR solves some more issues that began with https://github.com/joomla/joomla-cms/commit/c52e0d2ea90b9a73ac224d757a23add064f94e00
- Incorrect language strings (install sql and manifest files)
- Wrong version (3.6, not 3.5) (in manifest files)
- Wrong SQL dates for postgresql and sqlazure
- Other minor issues
#### Testing Instructions

Code review.

### CODE REVIEW

File: installation/sql/mysql/joomla.sql [hunk 1]
```diff
@@ -611,8 +611,8 @@ INSERT INTO `#__extensions` (`extension_id`, `name`, `type`, `element`, `folder`
 (452, 'plg_system_updatenotification', 'plugin', 'updatenotification', 'system', 0, 1, 1, 0, '', '', '', '', 0, '0000-00-00 00:00:00', 0, 0),
 (453, 'plg_editors-xtd_module', 'plugin', 'module', 'editors-xtd', 0, 1, 1, 0, '', '', '', '', 0, '0000-00-00 00:00:00', 0, 0),
 (454, 'plg_system_stats', 'plugin', 'stats', 'system', 0, 1, 1, 0, '', '', '', '', 0, '0000-00-00 00:00:00', 0, 0),
-(455, 'plg_installer_packageinstaller', 'plugin', 'packageinstaller', 'installer', 0, 1, 1, 1, '{"name":"plg_installer_packageinstaller","type":"plugin","creationDate":"07.05.2016","author":"Joomla","copyright":"Copyright (C) 2016 Open Source Matters. All rights reserved.","authorEmail":"admin@joomla.org","authorUrl":"www.joomla.org","version":"1.0","description":"PLG_PACKAGE_INSTALLER_PLUGIN_XML_DESCRIPTION","group":"","filename":"packageinstaller"}', '{}', '', '', 0, '0000-00-00 00:00:00', 1, 0),
```
Comment (mbabker) [inline installation/sql/mysql/joomla.sql]: Actually, the manifest cache's should be removed in full from the `joomla.sql` files (we populate them in the install app the same way they get refreshed on updates, so this is just extra static cruft that can be forgotten about).

We purposefully took them out a couple years ago for that reason (and the fact that we were having to set the CMS version number in here too on every bump).

Comment (andrepereiradasilva) [inline installation/sql/mysql/joomla.sql]: Joomla refresh the manifest on install? Didn't know that. Nice.

To be clear, you're talking just about the `{"name":"plg_installer_packageinstaller","type":"plugin","creationDate":"07.05.2016","author":"Joomla","copyright":"Copyright (C) 2016 Open Source Matters. All rights reserved.","authorEmail":"admin@joomla.org","authorUrl":"www.joomla.org","version":"1.0","description":"PLG_PACKAGE_INSTALLER_PLUGIN_XML_DESCRIPTION","group":"","filename":"packageinstaller"}` part, not the all SQL,  right?

Comment (mbabker) [inline installation/sql/mysql/joomla.sql]: Right.  The `manifest_cache` field's contents shouldn't be included in the SQL files.

And [right here](https://github.com/joomla/joomla-cms/blob/staging/installation/model/database.php#L602-L631) in the installer is where the magic is happening.  Just before the magic that apparently language distro's can use is processed.

Comment (andrepereiradasilva) [inline installation/sql/mysql/joomla.sql]: thabks! just updated this PR to remove the manifest cache

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on d6fd8e556941e3f1020a305973d4aaa5a5aae41e<br /><br />Great!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10296">issues.joomla.org/joomla-cms/10296</a>.</sub>

**Comment by roland-d** [comment]
@andrepereiradasilva We are working on the same thing. PLT actually decided to split the url/folder installer in two separate ones. There will be merge conflicts so I propose I merge your changes into the code I am writing.

**Comment by brianteeman** [comment]
Closing here as @roland-d will merge this directly into his own PR<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10296">issues.joomla.org/joomla-cms/10296</a>.</sub>

**Comment by andrepereiradasilva** [comment]
ok then.

**Pull request closed without merging**